### PR TITLE
First styling

### PR DESCRIPTION
**Pull request by Saskiazinha**

### CODE REVIEW

File: frontend/src/postsCommons/Post.js [hunk 1]
```diff
@@ -1,17 +1,131 @@
...
+    grid-row: 1;
+    grid-column: 3 / 3;
+
+    justify-self: end;
+  }
+`;
+
+const NameStyling = styled.h4`
+  grid-row: 1;
+  grid-column: 2 / 2;
```
Comment (benjamin-ranft) [inline frontend/src/postsCommons/Post.js:106]: This looks interesting, what does the 2 / 2 mean exactly? 👍

**Review by benjamin-ranft — APPROVED** [review]
Nice work! Keep it up 🥇

File: frontend/src/postsCommons/Post.js [hunk 2]
```diff
@@ -1,17 +1,131 @@
...
+  grid-template-rows: 38px 1fr;
+  grid-template-columns: repeat(3, minmax(0, 1fr));
+  border-radius: 20px;
+  box-shadow: 3px 3px 3px #95b0b4;
+  background-color: var(--turquoise-bright);
+  padding: var(--size-s);
+
+  button {
+    grid-row: 1;
+    grid-column: 3 / 3;
```
Comment (Saskiazinha) [inline frontend/src/postsCommons/Post.js:98]:
```suggestion
    grid-column: 3;
```

File: frontend/src/postsCommons/Post.js (hunk 1 above)
Comment (Saskiazinha) [reply frontend/src/postsCommons/Post.js:106]: Actually I just tried without the second 2 behind and it works as well. I was not sure if its needed to also define the end of the row when you just want to have it in one cell.  But works without, thanks for the hint!

Comment (Saskiazinha) [inline frontend/src/postsCommons/Post.js:106]:
```suggestion
  grid-column: 2;
```

File: frontend/src/postsCommons/Post.js [hunk 3]
```diff
@@ -1,17 +1,131 @@
...
+  align-self: center;
+  justify-self: center;
+  font-size: 1.1em;
+  font-weight: bold;
+  color: var(--turquoise-main);
+  letter-spacing: 0.2em;
+`;
+
+const ContentStyling = styled.div`
+  grid-row: 2 / 2;
```
Comment (Saskiazinha) [inline frontend/src/postsCommons/Post.js:116]:
```suggestion
  grid-row: 2 ;
```

**Pull request merged**